statusHandler.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+)
+
+type statusHandler struct {
+	workers []string
+}
+
+type status struct {
+	WorkerID int    `json:"worker_id"`
+	Status   string `json:"status"`
+}
+
+type allStatus []status
+
+func (h *statusHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// Make call out to get worker 2 status
+	Info.Println("Getting Worker Status...")
+	var allStat = allStatus{}
```

Comment:
why not just use a slice?  No need for this new type

Reply:
agree, this was dumb